src/main/resources/templates/user-frontpage.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+
+<head th:fragment="head(title)">
+    <title th:text="${title}">Frontpage</title>
```

Comment:
Why is there a fragment in the user-frontpage.html? This code looks to be for defining a fragment and not using a fragment? Should a fragment be used? Unless there's any intend with the fragment it could be changed to have the same structure as other existing html pages in the project